asciidoc-release-notes/adoc/release-notes.adoc

```diff
@@ -0,0 +1,25 @@
+include::attributes.adoc[]
+include::entities_all.adoc[]
+include::entities_product.adoc[]
+
+[#rnotes]
+= {rnotes}
+
+include::abstract.adoc[]
+
+include::about-rn.adoc[]
+
+// ===================================================================
+
+// START ARCH-INDEPENDENT
+[#allArch]
+== General Features & Fixes
```

Comment:
The ID and the title here don't fit very well together. I already updated that in the 15 SP3 RN, with my solution there being to remove this section head altogether because it just served to make the structure of the document more complicated without much gain.

In the interest of full disclosure, the story behind this section--make of it what you will:
* In the old days, FATE had categories "ReleaseNotes12:ArchIndependent:..." (and corresponding categories for POWER, IBM Z, and AArch64)
* Frank changed that the ID to AllArch when he set up the 12 SP5 and 15 SP2 RN.
* The title comes from a reorganization idea that I had (that is also reflected in https://confluence.suse.com/x/-IBMDQ ) where we had subsections "Features & Fixes" and "Known Issues & Workarounds" in each of the sections (Security with Security-Features and Security-Issues, Storage with Storage-Features and Storage-Issues, etc.). This was in part informed by RHEL and Ubuntu release notes.
* However, Frank found that solution with the many subsections to be unworkable in a manually updated document. He decided to inverse the structure and make one big section "Features & Fixes" (named AllArch, containing AllArch-Security, AllArch-Storage, etc.) and one big section "Known Issues & Workarounds" that had no further subsections and would just contain all the issues.
* After editing the 15 SP2 RN for a while, I noticed we had _nothing_ in the "Known Issues & Workarounds" section, since everyone (including Frank and me) had focused their efforts on the "Features & Fixes" section and all the known issues were listed in the "Features & Fixes" section. I thus abolished the "Known Issues & Workarounds" entirely, and all that was left was this section title.
* For 15 SP3, I removed the section title as well, so the structure became a bit flatter.

... So there. I think some way to tag known issues would still be a good idea. However, our attempts so far of sectioning them off seem to have failed. Maybe you have better ideas of how to do that/enforce that in the future.